eos/effects/modulebonusnetworkedsensorarray.py

```diff
@@ -4,13 +4,18 @@
 # Module: Networked Sensor Array
 type = "active"
 def handler(fit, src, context):
-    fit.ship.multiplyItemAttr("maxTargetRange", src.getModifiedItemAttr("maxTargetRangeMultiplier"), stackingPenalties=True, penaltyGroup="postMul")
     fit.ship.boostItemAttr("scanResolution", src.getModifiedItemAttr("scanResolutionBonus"), stackingPenalties=True)
 
     for scanType in ('Magnetometric', 'Ladar', 'Gravimetric', 'Radar'):
         fit.ship.boostItemAttr("scan{}Strength".format(scanType),
                                src.getModifiedItemAttr("scan{}StrengthPercent".format(scanType)),
                                stackingPenalties=True)
+                               
+    for fighter in fit.fighters:
+        for scanType in ('Magnetometric', 'Ladar', 'Gravimetric', 'Radar'):
+            fighter.boostItemAttr("scan{}Strength".format(scanType),
+                               src.getModifiedItemAttr("scan{}StrengthPercent".format(scanType)),
+                               stackingPenalties=True)
```

Comment:
>This basically accomplishes the same thing, but it's the construct uses in nearly all other effect files.

Oh the second half of this statement is _such_ a lie. :D

Still, it's better to use it.